packages/s/statosio.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "statosio.js",
+  "description": "Generate simple charts, in a fast and reliable way.",
+  "filename": "statosio.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/a6b8/statosio.js"
+  },
+  "keywords": [
+    "d3",
+    "d3js",
+    "statosio",
+    "statosiojs",
+    "javascript",
+    "svg",
+    "bar-chart",
+    "point-chart",
+    "bar-stacked-chart"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/a6b8/statosio.js.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js"
```
Open wildcards are dangerous, let's restrict this